nft/payment_obligation.go

```diff
@@ -2,8 +2,42 @@ package nft
 
 import (
 	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+
+	"github.com/centrifuge/go-centrifuge/utils"
 )
 
+// TokenIDLength is the length of an NFT token ID
+const TokenIDLength = 32
+
+// TokenID is uint256 in Solidity (256 bits | max value is 2^256-1)
+// tokenID should be random 32 bytes (32 byte = 256 bits)
+type TokenID [TokenIDLength]byte
+
+// NewTokenID returns a new random TokenID
+func NewTokenID() TokenID {
+	var tid [TokenIDLength]byte
+	copy(tid[:], utils.RandomSlice(TokenIDLength))
+	return tid
+}
+
+// BigInt converts tokenID to big int
+func (t TokenID) BigInt() *big.Int {
+	return utils.ByteSliceToBigInt(t[:])
+}
+
+// URI gets the URI for this token
+func (t TokenID) URI() string {
+	// TODO please fix this
+	return "http:=//www.centrifuge.io/DUMMY_URI_SERVICE"
```

Comment:
Will you fix this? I think here we should read from the ethereum contract to get the URI. This should be part of the NFT epic.